### DISCUSSION TITLE
Why do we have a `@` symbol as a prefix to keys in the documentation?

### DISCUSSION THREAD

**Discussion by yagudaev** [react-native-async-storage/async-storage#592] [Q&A]
Why does the documentation use the `@` symbol as the prefix for keys?

Is there some standard around it?

**Comment by tido64** [comment]
There is no standard. You just need to make sure that your keys are unique. Remember that the storage is the same for the whole app. If you have many features in there, it's easier to get a conflict if you're not careful.

**Reply by yagudaev** [reply]
So the `@` is just trying to signify keys should be unique? Can we just add that as a tip in the documentation?

**Reply by tido64** [reply]
I don't think the use of `@` signifies anything. Maybe @Krizzu knows the history behind? Regardless, I think it's irrelevant as long as you use unique keys.

Adding a tip to the documentation sounds like a good idea though.
